src/main/java/com/epam/code/mie/library/services/AuthorService.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.code.mie.library.services;
+
+import com.epam.code.mie.library.repositories.AuthorRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AuthorService {
+
+    @Autowired
```

Comment:
Do not use Autowired annotation, use constrauctor injection along with @RequiredArgConstructor lombook annotation